docs/prerequisites.rst

```diff
@@ -81,6 +81,7 @@ With Ubuntu 16.04, create a systemd init script at
 
 and start it with: ::
 
+  sudo chmod 600 var/lib/mongodb/
```

Comment:
Shouldn't this be `/var/lib/mongodb` (with a leading slash)?

And, if we are updating information on installing Mongo, should we update the key server and apt repository information to the current values?